src/fidasim.f90

```diff
@@ -2181,9 +2172,77 @@ subroutine read_inputs
     if(error) then
         stop
     endif
-
+    
+    !+ Identify the transition from lambdamax and lambdamin    
+    call identify_transition(n_stark, stark_pi, stark_sigma, &
+                                stark_intens, stark_wavel, line_lambda0)
+  
 end subroutine read_inputs
 
+subroutine identify_transition(n_stark, stark_pi, stark_sigma, &
+                                stark_intens, stark_wavel, line_lambda0)
+    !+ Determines the type of transition from user defined inputs
+    
+    ! real(Float64), intent(in)                                :: lambda_max, lambda_min
+    !+ maximum and minimum wavelength from user defined inputs
+    integer, intent(out)                                     :: n_stark
+    integer, dimension(:), allocatable, intent(out)          :: stark_pi, stark_sigma
+    real(FLoat64), dimension(:), allocatable, intent(out)    :: stark_intens, stark_wavel
+    real(Float64), dimension(3), intent(out)                 :: line_lambda0
+
+    if (inputs%lambdamin > 620.0d0 .and. inputs%lambdamax < 680.0d0 ) then
+        !+ Assigns stark variables to balmer alpha transition
+        n_stark = 15
+        line_lambda0 = [ 656.28d0, 656.104d0, 656.045d0 ]
+         !+"Tritium Diagnostics by Balmer-alpha emission" CH Skinner 1993
```

Comment:
just fyi dont use `!+` for comments within the code. Comments marked with `!+` get parsed by the documentation generator so we dont want random comments in the FIDASIM website